### ISSUE TITLE
Cron.parseAndBuild does not properly resolve for JS and Native

### ISSUE THREAD

**Issue by mgroth0** [Scogun/kcron-common#30]
It resolves properly for android and jvm

But on JS and Native, it says it "requires dependency" on the core module.

**Comment by Scogun** [comment]
Thank you for highlighting a problem.
Do you use the last version `0.10.5` in both cases?

**Comment by Scogun** [comment]
So, I tested the library for `mingwX64`:
```kotlin
plugins {
    kotlin("multiplatform") version "1.9.23"
}

repositories {
    mavenCentral()
}

kotlin {
    mingwX64 {
        binaries.executable()
    }
    sourceSets {
        val mingwX64Main by getting {
            dependencies {
                implementation("com.ucasoft.kcron:kcron-common:0.10.5")
            }
        }
    }
}
```
```kotlin
import com.ucasoft.kcron.Cron

fun main() {
    println(Cron.parseAndBuild("* 12 ? * *").expression)
}
```
Works as expected. You don't need `core` module here.
Could you confirm or clarify?

**Comment by Scogun** [comment]
Yes, I can confirm that with Kotlin `2.0.0-Beta4` version there is the issue.
Looks like it has problem with dependencies resolving. I will create a ticket on JetBrains board.
Thank you!

**Comment by Scogun** [comment]
Here is a link on JetBrains board: [KT-66707](https://youtrack.jetbrains.com/issue/KT-66707) Dependency library cannot be resolve

**Comment by Scogun** [comment]
Hello @mgroth0!
I just released new version of Kcron - 0.13.2
The issue is fixed via workaround, so I don't close the issue until [KT-66707](https://youtrack.jetbrains.com/issue/KT-66707) is not closed.
